src/Documentation/grains/index.md

```diff
@@ -65,10 +65,49 @@ public class PlayerGrain : Grain, IPlayerGrain
 }
 ```
 
+Grains can have two interfaces and one implementation.
+
+```csharp
+namespace UserNotification.Host
+{
+    public class MatchMakingGrain : Grain, IGameGrain, IPlayerDisconnect
```

Comment:
Since we already said that this code is an excerpt from the Presence sample, we should change it here arbitrarily. I think a better way to show a) a grain class implementing multiple interfaces and b) multiple grain classes implementing the same interface is to have separate subsection for those cases with different code snippets and not mix them into the basic case.